scripts/utility.cake

```diff
@@ -29,7 +29,10 @@ void CopyFiles(IEnumerable<string> files, string targetDirectory, bool clean = t
     }
     foreach (var file in files)
     {
-        CopyFile(file, targetDirectory + "/" + System.IO.Path.GetFileName(file));
+        if (FileExists(file))
```

Comment:
Let's revert this to identify problems early

Reply:
Ok.